# ğŸ–¥ï¸ High-performance computing (HPC) skills

## ğŸ“œ SLURM & bash scripting

In my master's thesis "Muurahaisten RNA -viruskirjon kartoittaminen"(Mapping of the ant RNA virome)(Not publicly available but can be shared upon request) 
I used CSCâ€™s Puhti supercomputing environment to perform analysis for a large RNA-seq dataset.
I executed multiple batch jobs using SLURM and Bash scripting, processing transcriptomic data from 66 ant species with a bioinformatic pipeline desined for virus and bateria detection, called Lazypipe.
This involved running automated workflows that performed quality control, de novo assembly, and annotation of RNA-seq and reference genome data efficiently in parallel computing environments.

## âœ”ï¸ Through this work, I became comfortable with:

- Navigating Linux-based systems

- Writing and submitting batch scripts

- Managing job dependencies and computational resources

- Monitoring and troubleshooting jobs in large-scale analyses

## ğŸ“ Puhti HPC example scripts

This folder contains example scripts demonstrating batch jobs and analyses I have performed on the Puhti supercomputer. I share these only for demonstration purposes to showcase my skills for future employers. If you are looking for help with your scripts, these shared scripts probably won't be helpful to you.

**Scripts for further analysis for lazypipe results coming soon**


## ğŸ”— More about Lazypipe:
Ilya Plyusnin, Ravi Kant, Anne J JÃ¤Ã¤skelÃ¤inen, Tarja Sironen, Liisa Holm, Olli Vapalahti, Teemu Smura, Novel NGS pipeline for virus discovery from a wide spectrum of hosts and sample types, Virus Evolution, Volume 6, Issue 2, July 2020, veaa091, https://doi.org/10.1093/ve/veaa091


