# High-performance computing (HPC) skills

## SLURM & bash scripting

In my master's thesis "Muurahaisten RNA -viruskirjon kartoittaminen"(Mapping of the ant RNA virome)(Not publicly available but can be shared upon request) 
I used CSC’s Puhti supercomputing environment to run large-scale RNA-seq analyses.
I executed multiple batch jobs using SLURM and Bash scripting, processing transcriptomic data from 66 ant species with a bioinformatic pipeline desined for virus and bateria detection, called Lazypipe.
This involved running automated workflows that performed quality control, de novo assembly, and annotation of RNA-seq and reference genome data efficiently in parallel computing environments.

** Through this work, I became comfortable with: **





Navigating Linux-based systems

Writing and submitting batch scripts

Managing job dependencies and computational resources

Monitoring and troubleshooting jobs in large-scale analyses


More about Lazypipe:
Ilya Plyusnin, Ravi Kant, Anne J Jääskeläinen, Tarja Sironen, Liisa Holm, Olli Vapalahti, Teemu Smura, Novel NGS pipeline for virus discovery from a wide spectrum of hosts and sample types, Virus Evolution, Volume 6, Issue 2, July 2020, veaa091, https://doi.org/10.1093/ve/veaa091
